admin/metrics/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='OSFStatistic',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('users', models.IntegerField(default=0, verbose_name=b'OSF users')),
+                ('delta_users', models.IntegerField(default=0)),
+                ('unregistered_users', models.IntegerField(default=0, verbose_name=b'Unregistered users')),
+                ('projects', models.IntegerField(default=0, verbose_name=b'Number of projects')),
+                ('delta_projects', models.IntegerField(default=0)),
+                ('public_projects', models.IntegerField(default=0, verbose_name=b'Number of public projects')),
+                ('delta_public_projects', models.IntegerField(default=0)),
+                ('registered_projects', models.IntegerField(default=0, verbose_name=b'Number of projects registered')),
+                ('delta_registered_projects', models.IntegerField(default=0)),
+                ('date', models.DateTimeField(default=None)),
```

Comment:
It is apparent that we don't have the same communications channels, specifically lunch talks and what other people are working on. As much as I would like to work from home, or while traveling, I am beginning to realize just how tough it is to keep up-to-date remotely. So let me fill you in.

I will post the link to the lunch talk later if there is one, but right now Fitz is looking into keen.io for more of the transaction based metrics. An we could certainly integrate that into Admin interface. 

That doesn't quite fully explain the mismatch. This comes from conversations with Sara and Matt Spitzer on how we need to drastically overhaul how we track metrics, and how we want to be much more focused on how users are using the site, including reading projects and not just making another project. This is where keen.io comes in.

It is true that the Jira ticket won't be covered by this, nor will it cover all the metrics that community wants. I will discuss this today to see if they would rather this be moved to a proof-of-concept' ticket while the rest of metrics is nailed down.